build.gradle

```
@@ -66,7 +66,7 @@ dependencies {
 }
 
 shadowJar {
-    archiveFileName = 'addressbook.jar'
+    archiveFileName = 'dukeTheMarket.jar'
```

Hmm I'm not exactly sure about the naming convention for JAR files as the module does not specify it.
Since AddressBook is changed to addressbook, perhaps making everything lower case might be better as following the camelCase convention looks slightly weird for file names.
```suggestion
    archiveFileName = 'dukethemarket.jar'
```

---

Yeah I also wasn't sure what naming convention to use since they didn't specify it. I've asked our TA about it, let's see what he says.

---

Prof replied to my [forum post](https://github.com/nus-cs2103-AY2223S1/forum/issues/357) about the naming convention. Looks like there is no convention we need to follow. So I suggest we go with the current camelCase option. i.e. `dukeTheMarket.jar` 

I'd actually prefer UpperCamelCase since the filename looks better in "Title case", but when running the JAR file on the console I think it's a little more troublesome to type an uppercase letter 😆